### PR TITLE
Updates to Urban Cooling UG chapter

### PR DESCRIPTION
**Pull request by jagoldstein**
substantial edits made to "urban_cooling_model.rst" and "ucm_appendix_table.csv" to improve grammar, readability, and language consistency

### CODE REVIEW

**Comment by jagoldstein** [comment]
I reviewed the HTML and CSV files. They both look good and this PR is ready to be merged.

File: source/urban_cooling_model.rst
Comment (emlys) [inline source/urban_cooling_model.rst:210]: I kind of like using the unicode degree symbol (°) where possible. It's probably fine but I wonder if "degC", especially run together as one word, might be confusing to users who aren't native English speakers. Spelling out fully and/or using SI conventions may be more universally understandable.

File: source/urban_cooling_model.rst
Comment (emlys) [inline source/urban_cooling_model.rst:212]: The UI does set a default of 2000 for this input [here](https://github.com/natcap/invest/blob/01a3ddeac94d60afd124542b15503fdf916abb0f/src/natcap/invest/ui/urban_cooling_model.py#L75) .However, I'm not sure about the concept of a "default" value to begin with. Here are some problems I see with it: 
- When you save the inputs from the last model run, any defaults are overridden. As far as I know there isn't a way to "reset defaults" in the current UI.
- This is because a "default" is handled entirely on the UI side: it's simply a string that's put into the input field when it first loads.
- There is no representation of a "default value" in the `ARGS_SPEC`, and I'm not convinced that there should be.
- Because it's not represented in the `ARGS_SPEC`, it will not carry over to the workbench UI.
- All a "default" really means is a commonly-accepted value that may be suitable for a lot of scenarios. Officially encoding this into the model spec would introduce some questions with unclear answers: How common does a value have to be, to get the designation of "default"? How do we decide which inputs get defaults? Are we accidentally misleading users or promoting certain papers/methods by giving extra weight to a "default" value?

All that is to say that I don't really like the term "default", and I might say something like "A common value is 2000 m (citation)" instead. @phargogh do you have any thoughts about defaults?

File: source/urban_cooling_model.rst
Comment (emlys) [inline source/urban_cooling_model.rst:214]: Hooray for the oxford comma! I want this in our style guide 🤓

**Review by emlys — CHANGES_REQUESTED** [review]
Thanks for this @jagoldstein! Definitely a good improvement and all our work will seem more reputable when we follow good writing style guidelines. 

To avoid us duplicating efforts, I might suggest holding off on doing this for other models for now. Urban Cooling is pretty unique. But some other model user guides have entire sections that are repeated word-for-word in other models (e.g. https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest/ndr.html#appendix-data-sources) or reuse common phrases/explanations ("Make sure that there is ample disk space and that write permissions are correct.", ".csv (Comma Separated Values)"). All of these I'd like to put in one centralized file where we go over common inputs, outputs, file type basics, etc., and that would happen before the translations.
There is also a lot of overlapping text in the `"about"` property of each model `ARGS_SPEC` that will need to be merged with user's guide descriptions (again to reduce translation costs, and so we can automatically generate those portions of the user's guide). So it might make sense to do this kind of editing for other models after those changes happen.

File: source/urban_cooling_model.rst
Comment (davemfish) [reply source/urban_cooling_model.rst:15]: I think the "UHI" acronym needs to be spelled out on it's first use.

Comment (davemfish) [reply source/urban_cooling_model.rst:15]: My naive thinking on energy balance is just "heat going in vs heat going out". But sentence could also read:

"...is caused by 1) a change in the thermal properties of materials used in urban areas (e.g. concrete, asphalt), which store more heat, and 2) the reduction of the cooling effect of vegetation (through shade and evapotranspiration).

Comment (jagoldstein) [reply source/urban_cooling_model.rst:15]: It's just a general term used in thermodynamics, but I agree that it may be overly technical for this sentence
```suggestion
UHIs affect many cities around the world, with major consequences for human health and wellbeing: high mortality or morbidity during heat waves, high A/C consumption, and reduced comfort or work productivity. The UHI effect, i.e. the difference between rural and urban temperatures, is a result of the unique characteristics of cities due to two main factors: the thermal properties of materials used in urban areas (e.g. concrete, asphalt), which store more heat, and the reduction of the cooling effect (through shade and evapotranspiration) of vegetation.

```

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:28]: Yes, this is an intentional change to more precise language. See Section 2.2 from [Zardo et al., 2017](https://doi.org/10.1016/j.ecoser.2017.06.016), "_trees with canopy equal or higher than two meters_". Nice catch though, thanks for asking!

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:56]: Thanks for bringing this up and for being so detail focused. I don't have a preference for space or not, but could easily be convinced that using one is best. Looking through the UG, I see it both ways, often within the same model chapter, so YES, as long as it's consistent throughout, I'd be happy. +1 for this being 
> part of a different, style-focused PR that touches all the models

File: source/urban_cooling_model.rst
Comment (phargogh) [reply source/urban_cooling_model.rst:212]: > All that is to say that I don't really like the term "default", and I might say something like "A common value is 2000 m (citation)" instead.

Yes, I agree that "default" implies that the value is a fallback, which isn't quite correct.  I think "recommended" value is maybe closer to what we really mean ... it's more like a "this value should be sufficient for most use cases, but we want you to be able to tweak it if you know what you're doing".

RE: citations, these would be great to include with a recommended value where available!  But this should be able to handle 0+ citations.  A concrete use case for this is that RouteDEM allows the user to provide a band index (recommended value: 1).  DelineateIt has a checkbox marked as True (whether to skip invalid geometry).  No citations needed there.

> There is no representation of a "default value" in the ARGS_SPEC, and I'm not convinced that there should be.

This is true!  We've been shoehorning a default value in to the UI files or just using the parameter set files to represent what might be better represented as default/recommended values.

I have 2 reasons why I think we should seriously consider adding a recommended value to the ARGS_SPEC for non-file inputs:
1. Our science staff sometimes ask for them!  At times the request is better represented as part of a parameter stack, but there do seem to be some 'expert opinion' cases where a natcap-scientist recommended value might be good enough for most.
2. For cases where the model has a recommended or default value anyways (urban cooling example within `execute`), it'd be better to have a single source of truth for that value by referring back to the ARGS_SPEC.  This could also enable us to have a more consistent user experience with recommended/default values across models, too.

If we were to have ARGS_SPEC be the source of truth for recommended values, it would also solve the other concerns you mentioned about values not being available to the workbench and would allow for an individual value to be reset to its original recommended/default value.

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:210]: I agree. I somewhat lazily choose 'degC' here this time to make it short and consistent because there were a variety of spellings including an incorrect one ("Celcius"; plus a couple "centigrade"s). I think the unicode symbol (°) would be better than 'degC' for the reasons @emlys describes. SI conventions and/or fully spelling out units would be better too (and more inclusive). Again, I could be convinced of almost any way here, as long as we establish a standard and stick to it.

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:212]: Great points @emlys , especially about maybe just using a different wording, like "common value" or "recommended starting value" rather than "default".
Specific to this case, please see [this Forums post](https://community.naturalcapitalproject.org/t/urban-cooling-queries-about-parameterization/915) from April 2020 where @royremme wrote,
> "I suggest starting with the 500m value in the appendix that is based on the first studies we have used the model in. The 2000m is an artifact from an older version of the user guide, we need to update that".
That doesn't quite explain my "600m", but I believe this value came from what @newtpatrol and @chrisnootenboom have used most recently for the Twin Cities and what we recommended in the data packet we used for Gretchen's class.

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:214]: Yes, Oxford comma love! I cringe at lists without it :fearful:.

BUT for the translations, be aware that, in my experience, the Oxford comma is (universally?) absent from the Spanish language.

Comment (emlys) [reply source/urban_cooling_model.rst:214]: Oh, good to know!

Comment (phargogh) [reply source/urban_cooling_model.rst:214]: That is interesting to know about the Spanish language!

When all of the text here is translated, it'll be up to the translator to translate each chunk (in this case, it'll most likely be a paragraph) in a way that makes sense for the target language, and the text will be swapped in paragraph-by-paragraph.  So for us, we should for sure make decisions about English grammar or style that make sense for English and then do the same for Spanish when we translate it into Spanish.  If the two paragraphs end up having a _slightly_ different arrangement of commas, that'll be ok!

File: source/urban_cooling_model.rst
Comment (phargogh) [reply source/urban_cooling_model.rst:212]: >>    "I suggest starting with the 500m value in the appendix that is based on the first studies we have used the model in. The 2000m is an artifact from an older version of the user guide, we need to update that".
>
>That doesn't quite explain my "600m", but I believe this value came from what @newtpatrol and @chrisnootenboom have used most recently for the Twin Cities and what we recommended in the data packet we used for Gretchen's class.

@jagoldstein is it your impression that this value will be specific to the site, or do you think this will be a pretty common value across study areas?  If it's for the sake of demonstration, I'd think this might be better-suited to a parameter set (demonstrating this numeric value in the context of other datasets that made sense for that analysis) than a standalone recommended value.  What do you think about that?

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:15]: I agree that acronyms should be spelled out on first use and then not again. The first occurrence of "urban heat island" is in the 'Summay' section that precedes the 'Introduction', so that is where I first defined "UHI".

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:212]: @phargogh great points and questions. It is my impression that there can be relatively high uncertainty in the max air blending distance, as described by @chrisnootenboom in [this Forums post](https://community.naturalcapitalproject.org/t/air-temperature-maximum-blending-distance-uncertainty/1795/3). Chris says that 600m comes from "Shatz and Kucharik papers" (not cited in UG). The model was tested in the Twin Cities and Paris, France before release, but _not_ in significantly different locations, like the tropics. I think your idea of a parameter set sounds like it could make more sense than "recommended" or "default" values. I should defer to Chris or Eric though as I am far from an expert on this.

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:57]: I'm convinced @emlys . I'm going to recommend [this standard](https://developers.google.com/style/units-of-measure) for our [style guide.](https://docs.google.com/document/d/1BHwHDu_I-x0s_2GsbUb4rfVmXMkl7kl97sx2suBTLh8/edit?usp=sharing)

File: source/urban_cooling_model.rst
Comment (chrisnootenboom) [reply source/urban_cooling_model.rst]: @jagoldstein @phargogh I just caught up on this thread and can confirm: the 500m value was derived from regression modeling in the Schatz and Kucharik paper (https://doi.org/10.1175/JAMC-D-14-0107.1) while the 600m value came from similar regression modeling in our golf paper (https://doi.org/10.1016/j.landurbplan.2020.104022). Users should be aware of this limitation when applying the model to other climate zones, but there just aren't many data sources on this value out there. 
It is unclear whether the parameter will be hyper-specific to an individual city--air flow dynamics should be consistent globally, but our current calibration studies in the Twin Cities and Paris result in a range of values from as low as 260m to as high as 1600m. So the jury is definitely out on a single 'default' value even in cities in temperate climates!
That being said, I would reference the two papers I linked above in the users guide if possible, since 500 or 600m is a reasonable first pass for the model.

File: source/urban_cooling_model.rst
Comment (emlys) [reply source/urban_cooling_model.rst]: Neat! That google style guide looks perfect.

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst]: Thanks for this info @chrisnootenboom ! I adjusted the language accordingly and added these two sources to the References section.

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:207]: I agree about needing those spaces before the unit. >2 ha is correct. The change to 'greater than or equal to' was for the 2 m tree canopy height, not the 2 ha green space area.

**Comment by davemfish** [comment]
@jagoldstein I haven't looked at this lately but it looks like Emily has given a pretty thorough review. Let me know if there is anything specific I can weigh-in on, otherwise I think Emily's review is probably sufficient and maybe I can be off-the-hook for reviewing this one?

**Comment by jagoldstein** [comment]
You are off the hook @davemfish ! @emlys is all over it. I just wasn't sure of the best way to proceed after committing further changes following the first round of reviews. One thing I learned is NOT to resolve comments (because it seems I can not retrieve them after that).

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:132]: good catch... I'm going with `° C`, per your earlier suggestion and our current style guide

File: source/urban_cooling_model.rst
Comment (jagoldstein) [reply source/urban_cooling_model.rst:200]: `>2 ha`

**Comment by jagoldstein** [comment]
@emlys I believe I made all of the needed changes that you identified. Thanks again for being so thorough! I hope this PR is ready for merging now 🤞

**Review by emlys — APPROVED** [review]
Awesome thank you @jagoldstein for putting up with my picky suggestions! 😁

**Pull request merged**
